README.md

```diff
@@ -1,22 +1,43 @@
-# kubescape.io website
+# kubescape.io Website
 
-This repository contains the source files for the kubescape.io website.
+This repository contains the source files for the [kubescape.io](https://kubescape.io) website. The website is built using [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme.
 
-## Building the site
+## Table of Contents
 
-### Local testing
+- [Getting Started](#getting-started)
+- [Local Testing](#local-testing)
+- [Production](#production)
+- [Contributing](#contributing)
+- [License](#license)
```

Comment:
You didn't remove the license.  And you really don't need a table of contents in a 10 line file.